termux-exec.c

```diff
@@ -72,7 +72,12 @@ static char*const * remove_ld_preload(char*const * envp)
 	return envp;
 }
 
-int execve(const char* filename, char* const* argv, char* const* envp)
+
+int exec_wrapper(
```

Comment:
You should add `static`. We definitely don't want to pollute the global namespace.